scripts/scil_reorder_connectivity.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Re-order a connectivity matrix using a json file in a format such as:
+    {"temporal": [[1,3,5,7], [0,2,4,6]]}.
+The key is to identify the sub-network, the first list is for the
+column (x) and the second is for the row (y).
+
+The values refers to the coordinates in the matrix, but if the --labels_list
+parameter is used, the values will refers to the label which will be converted
+to the appropriate coordinates. This file must be the same as the one provided
+to the scil_decompose_connectivity.py
```

Comment:
Could you give a small example, like you did higher with coordinates?

Reply:
Once everything is merged, I will create an example/tutorial separately